Source/WebCore/css/StyleProperties.h

```diff
@@ -155,195 +140,26 @@ class StyleProperties : public RefCounted<StyleProperties> {
 
     bool propertyMatches(CSSPropertyID, const CSSValue*) const;
 
-    int findPropertyIndex(CSSPropertyID) const;
-    int findCustomPropertyIndex(StringView propertyName) const;
+    inline int findPropertyIndex(CSSPropertyID) const;
+    inline int findCustomPropertyIndex(StringView propertyName) const;
 
 protected:
-    StyleProperties(CSSParserMode mode, StylePropertiesType type)
-        : m_cssParserMode(mode)
-        , m_type(type)
-    { }
-
-    StyleProperties(CSSParserMode mode, unsigned immutableArraySize)
-        : m_cssParserMode(mode)
-        , m_type(ImmutablePropertiesType)
-        , m_arraySize(immutableArraySize)
-    { }
+    inline explicit StyleProperties(CSSParserMode);
+    inline StyleProperties(CSSParserMode, unsigned immutableArraySize);
 
     unsigned m_cssParserMode : 3;
-    mutable unsigned m_type : 2;
-    unsigned m_arraySize : 27 { 0 };
+    mutable unsigned m_isMutable : 1 { true };
+    unsigned m_arraySize : 28 { 0 };
```

Comment:
Why does this increase?

Reply:
These bitfields together take 32 bits. We don’t need 2 bits for `type` any more, just one byte for `isMutable`. Later maybe we should figure out the real maximum for `arraySize` and set that number of bits precisely. For now, though, as before, it’s just "the rest of the 32 bits".